marshal/consumer.go

```diff
@@ -853,7 +853,10 @@ func (c *Consumer) Commit(msg *Message) error {
 		return cl, ok
 	}()
 	if !ok {
-		return errors.New("Message not committed (partition claim expired).")
+		return fmt.Errorf(
```

Comment:
nit: Too many lines. You can do this in two lines and still obey the line length limit. (I'm allergic to gratuitous vertical space; hard to read when there's one word per line.)